dist/osel-search.js

```diff
@@ -41,6 +41,17 @@
         }, {});
         // ---------- variables setup end -----------
 
+          var observableWithPreTransformResult = function(preTransformResult, term) {
+              return rx.Observable.create(function (observer) {
+                  try {
+                      var result = preTransformResult.call(this, term);
+                          observer.onNext(result);
```

Comment:
just interested why "observer.onCompleted()" is not called here.

(I was subscribed to this as I made the android version :smile:)